core/src/repl/mod.rs

```diff
@@ -113,7 +113,12 @@ impl<EC: EvalCache> ReplImpl<EC> {
             resolved_ids: pending,
         } = import_resolution::strict::resolve_imports(t, self.vm.import_resolver_mut())?;
         for id in &pending {
-            dbg!(self.vm.import_resolver_mut().resolve_imports(*id)).unwrap();
```

Comment:
Ah, an `unwrap` without explaining why it's assumed to be ok...good example that this is a bad thing :upside_down_face: